frontend/src/staging/StagingBrowser.vue

```diff
@@ -243,8 +243,8 @@ export default class StagingBrowser extends Vue {
         });
     }
 
-    getFilesToShow(marked:boolean) {
-        if (marked) {
+    getFilesToShow() {
+        if (this.showMarked) {
             return this.filesToShow;
         }
         return this.filesToShow.filter(file => !file.marked);
```

Comment:
Wenn man es jetzt so sieht, würde ich es noch einmal anders machen. Nämlich als computed property
:
https://alligator.io/vuejs/typescript-class-components/#computed-properties

In dem Fall würde ich es dann einfach "showMarked" nennen, und entsprechend in der methode true/false zurückgeben anstatt das in der aktuellen Form im Template zu evaluieren.